component/identity/src/test/java/org/exoplatform/services/organization/TestOrganization.java

```diff
@@ -210,6 +211,42 @@ public void testFindGroupOfUser() {
         }
     }
 
+    public void testFindGroupsOfUserByKeyword() throws Exception
+    {
+        GroupHandler groupHandler = organizationService.getGroupHandler();
+        List<String> excludedGroupsTypes = new ArrayList<>(List.of("GTN_ROOT_GROUP"));
+        try {
+            Collection<Group> groups = groupHandler.findGroupsOfUserByKeyword("john","us",excludedGroupsTypes);
+            assertNotNull(groups);
+            assertEquals(1,groups.size());
+            assertEquals(1,groupHandler.findGroupsOfUserByKeyword("john","ad",excludedGroupsTypes).size());
+            assertEquals(1,groupHandler.findGroupsOfUserByKeyword("demo","us",excludedGroupsTypes).size());
+            assertEquals(0,groupHandler.findGroupsOfUserByKeyword("demo","ad",excludedGroupsTypes).size());
+            excludedGroupsTypes.add("GTN_GROUP_TYPE");
+            assertEquals(0,groupHandler.findGroupsOfUserByKeyword("john","ad",excludedGroupsTypes).size());
+        } catch (Exception e) {
+            fail();
```

Comment:
There is a throws in the method signature, thus no need to add a `try/catch` + `fail` instruction